tests/rt.sh

```diff
@@ -159,7 +159,7 @@ elif [[ $MACHINE_ID = gaea.* ]]; then
   ECFLOW_START=
   # DH* 20190717 temporary
   #DISKNM=/lustre/f2/pdata/ncep_shared/emc.nemspara/RT
-  DISKNM=/lustre/f2/pdata/esrl/gsd/gmtb/NEMSfv3gfs/RT
+  DISKNM=/lustre/f2/pdata/esrl/gsd/ufs/ufs-weather-model/RT
```

Comment:
How was the gaea baseline created? Is it same as cray? I am copying RT baseline from cray to gaea as requested by GFDL. If this is done, I can let GFDL know.

Reply:
This baseline was created on gaea using Intel 18. As far as I understand, wcoss_cray is still using Intel 16. Once we are using the same version of the compiler, we should check if the baselines are indeed identical and start copying from wcoss_cray if yes.

Reply:
OK, all model baseline under emc.nemspara on gaea was copied from cray. Now we need to copy baseline from cray to gaea first before we can verify that the current code can still generate same regression test baseline on gaea. Would you please add an option such as "GSD_BASELINE" so that we can still use the emc.nemspara under ncep_shared for wcoss cray baseline? We can combine the ENC baseline with GSD-BASELINE the verification is done.

Reply:
You mean I should use something like this:
```
DISKNM=/lustre/f2/pdata/ncep_shared/emc.nemspara/RT
#DISKNM_GSD=/lustre/f2/pdata/esrl/gsd/ufs/ufs-weather-model/RT
```
But I don't understand how the baseline from cray should work for gaea if the compiler is different? Or are you using the exact same Intel version on cray? Gaea is 18.0.3.222

In my opinion we cannot make this requested change until we are using the same version of the compiler.

Reply:
Note also that on cray we are not running rt.conf but rt_wcoss_cray.conf, which only runs IPD PROD. This unification of baselines is something for a future commit. Here I am just re-enabling the regression tests on gaea and at the same time showing that CCPP and IPD are b4b identical even with they cray compilers.